block-storage/src/main/scala/coop/rchain/blockstorage/InMemBlockStore.scala

```diff
@@ -17,9 +18,12 @@ class InMemBlockStore[F[_]] private ()(
     metricsF: Metrics[F]
 ) extends BlockStore[F] {
 
+  private implicit val metricsSource: MetricsSource =
+    MetricsSource(RSpaceMetricsSource, "block-storage.in-mem")
```

Comment:
Why the change of prefix from "block-store"?

Reply:
I don't think `RSpaceMetricsSource` should be used here

Reply:
this is `storage` module this different source should be used